plugin/sycl/common/partition_builder.h

```diff
@@ -0,0 +1,101 @@
+/*!
+ * Copyright 2017-2023 XGBoost contributors
```

Comment:
should it be 2024? Or just creation year

Reply:
fixed